packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -71,11 +71,12 @@ library ChainAssertions {
 
         if (_isProxy) {
             require(config.owner() == _cfg.finalSystemOwner());
-            require(config.overhead() == _cfg.gasPriceOracleOverhead());
-            require(config.scalar() == _cfg.gasPriceOracleScalar());
+            require(config.basefeeScalar() == _cfg.basefeeScalar());
+            require(config.blobbasefeeScalar() == _cfg.blobbasefeeScalar());
             require(config.batcherHash() == bytes32(uint256(uint160(_cfg.batchSenderAddress()))));
             require(config.gasLimit() == uint64(_cfg.l2GenesisBlockGasLimit()));
             require(config.unsafeBlockSigner() == _cfg.p2pSequencerAddress());
+            require(config.scalar() >> 248 == 1);
```

Comment:
Best practice with `require` statements IMO is to always use the string argument since it makes it a lot easier to identify where exactly an assertion failed. This is similar to what we did in the postCheck of superchain ops. I don't want to block this PR on modifying old code, but would at least like to start migrating new code to that. 

I like incrementing numbers of 100 so you can add new ones in between without adjusting all numbers, for example:
```solidity
require(config.unsafeBlockSigner() == _cfg.p2pSequencerAddress(), "600");
require(config.scalar() >> 248 == 1, "700");
```

then in the future, minimal diff to add a new check
```solidity
require(config.unsafeBlockSigner() == _cfg.p2pSequencerAddress(), "600");
require(myNewCheck, "650");
require(config.scalar() >> 248 == 1, "700");
```

Reply:
definitely agree with this, at some point we should do this